workers/social/lib/social/models/user/index.coffee

```diff
@@ -692,15 +693,17 @@ module.exports = class JUser extends jraphical.Module
 
         user.fetchAccount context, (err, account) ->
 
-          if err?
+          if err
 
             logout 'error fetching account', clientId, callback
 
           else
-
             # A valid session, a valid user attached to
             # it voila, scenario #2
-            callback null, { session, account }
+            session.lastAccess = lastAccess = new Date
```

Comment:
we dont need to update session each time we get a client for it.
either lets remove this code or lets add an in memory cache where we will update this once a while say 1 time per hour.